h/static/styles/app.scss

```diff
@@ -82,27 +82,82 @@ ol {
 
 /* The groups dropdown list. */
 
-$group-list-width: 225px;
+$group-list-width: 270px;
 
-.group-list .dropdown {
-  white-space: nowrap;
-}
+.group-list {
+  .dropdown {
+    white-space: nowrap;
+  }
 
-.group-list .dropdown-menu {
-  width: $group-list-width;
-}
+  .dropdown-menu {
+    width: $group-list-width;
 
-.group-list .dropdown-menu li {
-  @include pie-clearfix;
-}
+    .group-name {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      width: $group-list-width - 30px;
+    }
+  }
+
+  .group-item {
+    display: flex;
```

Comment:
> You're going to want to use the Compass mixins for this, or I think you'll miss out on IE10 support.

Ah, indeed. Actually Safari will be the bigger issue, they only dropped the prefix in Safari 9.

Has post-processing with [autoprefixer](https://github.com/postcss/autoprefixer) been considered for handling vendor prefixes automatically?